sphinx/directives/other.py

```diff
@@ -39,6 +39,7 @@ class TocTree(Directive):
     final_argument_whitespace = False
     option_spec = {
         'maxdepth': int,
+        'name': str,
```

Comment:
@ericholscher I guess it's time to release 1.3. How do you think about `:caption:`?